### ISSUE TITLE
Questions about using

### ISSUE THREAD

**Issue by timsofteng** [lambdalisue/fern.vim#200]
Hello.
I have few questions about fern using.

1. What option should I use to back cursor to previos window after closing fern by toggle?
2. How I can automatically close fern after opening file?

**Comment by lambdalisue** [comment]
1. Not sure what are you talking about
2. https://github.com/lambdalisue/fern.vim/issues/94

**Comment by timsofteng** [comment]
> 1. Not sure what are you talking about
> 2. #94

1. Imagine you open few windows inside vim. Then you try to open fern from one of them. And then you close fern by toggle option. In this way your cursor will locate in one specific window which is situated first from left side of terminal everytime. But I want back my cursor to window where I called fern. Are you understand?

2. I have tried this solution
`nmap <silent><buffer> <CR> <Plug>(fern-action-open)<ESC><BAR>:FernDo close<CR>`

I just put it into my .vimrc but it doesn't help. Fern still opened.
